src/App.js

```diff
@@ -1,30 +1,50 @@
 import React from 'react';
-
-import './App.css';
-
 import posts from './api/posts';
 import comments from './api/comments';
 import users from './api/users';
+import PostList from './PostList';
+
+const App = () => {
+  const preparedPosts = posts
+    .map((post) => {
+      const user = users.find(it => it.id === post.userId);
+
+      return {
+        ...post,
+        user,
+      };
+    }).map((post) => {
```

Comment:
why don't you do this action in the prev loop?

Reply:
my fault.
missed such an opportunity.
fixed.